src/browser/documentation/sidebar-guides/concepts.tsx

```diff
@@ -22,144 +22,167 @@ import React from 'react'
-      <li>Color "Person" nodes red</li>
+      <li>Add the label "Person" to the node you created for Emil.</li>
+      <li>Color the "Person" node red.</li>
     </ol>
-    <ul>
-      <li>A node can have zero or more labels</li>
-      <li>Labels do not have any properties</li>
-    </ul>
+    <br />
     <img src="./assets/images/labeled_node.png" className="img-responsive" />
+    <br />
+    <p className="paragraph">
+      <em>
+        <p>Key info:</p>
+        <ul>
+          <li>A node can have zero or more labels.</li>
+          <li>Labels do not have any properties.</li>
+        </ul>
+      </em>
+    </p>
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="forth">
     <h3>More Nodes</h3>
     <p className="lead">
-      Schema-free, nodes can have a mix of common and unique properties.
+      <em>
+        Neo4j is schema-free. Nodes can have a mix of common and unique
+        properties.
+      </em>
     </p>
     <p>
       Like any database, storing data in Neo4j can be as simple as adding more
-      records. We'll add a few more nodes:
+      nodes. Add a few more nodes and properties:
     </p>
     <ol>
-      <li>Emil has a klout score of 99</li>
-      <li>Johan, from Sweden, who is learning to surf</li>
-      <li>Ian, from England, who is an author</li>
-      <li>Rik, from Belgium, has a cat named Orval</li>
-      <li>Allison, from California, who surfs</li>
+      <li>Emil had a Klout score of 99.</li>
+      <li>Johan, from Sweden, who is learning to surf.</li>
+      <li>Ian, from England, who is an author.</li>
+      <li>Rik, from Belgium, who has a cat named Orval.</li>
+      <li>Allison, from the US, who surfs.</li>
     </ol>
-    <ul>
-      <li>Similar nodes can have different properties</li>
-      <li>Properties can be strings, numbers, or booleans</li>
-      <li>Neo4j can store billions of nodes</li>
-    </ul>
+    <br />
     <img src="./assets/images/more_nodes.png" className="img-responsive" />
+    <br />
+    <p className="paragraph">
+      <em>
+        <p>Key info:</p>
+        <ul>
+          <li>Similar nodes can have different properties.</li>
+          <li>Properties can be strings, numbers, or booleans.</li>
+          <li>Neo4j can store billions of nodes.</li>
+        </ul>
+      </em>
+    </p>
   </BuiltInGuideSidebarSlide>,
   <BuiltInGuideSidebarSlide key="fifth">
-    <h3>Consider Relationships</h3>
-    <p className="lead">Connect nodes in the graph</p>
-    <p className="summary">
+    <h3>Relationships</h3>
+    <p className="lead">
+      <em>Connect nodes in the graph</em>
+    </p>
+    <p>
       The real power of Neo4j is in connected data. To associate any two nodes,
-      add a Relationship which describes how the records are related.
+      add a relationship that describes how the records are related.
     </p>
-    <p>In our social graph, we simply say who KNOWS whom:</p>
+    <p>In our social graph, you can simply say who KNOWS whom:</p>
```

Comment:
In our social graph, you can simply say who knows (relationship type KNOWS) whom: